src/store/guestNameStore.js

```diff
@@ -20,86 +20,88 @@
  *
  */
 import Vue from 'vue'
+import { defineStore } from 'pinia'
+
+export const useGuestNameStore = defineStore('guestNameStore', {
```

Comment:
```suggestion
export const useGuestNameStore = defineStore('guestName', {
```

See `src/stores/bots.js` or `src/stores/settings.js` for example
* Pinia store should be moved to the different folder, as others, and renamed to `guestName` for simplicity
* Vuex store usages should be removed from the `src/store/storeConfig.js`
* Test coverage - 100%